syntax = "proto3";

package player;

service PlayerService {
  rpc PlayAudio (AudioRequest) returns (PlayResponse);
  rpc StopAudio (EmptyRequest) returns (StopResponse);
}

message AudioRequest {
  string audio_id = 1;
  string url = 2;
}

message PlayResponse {
  bool success = 1;
}

message StopResponse {
  bool success = 1;
}

message EmptyRequest {}
